src/common/AddressSwitcher.css

```diff
@@ -1,6 +1,6 @@
 .AddressSwitcher {
     height: 95px;
-    width: 360px;
+    width: 450px;
```

Comment:
Did you check with a 1200px wide window that the layout remains fine?

Reply:
Good point. Just checked, OK with this resolution.